sys/net/routing/nhdp/nhdp_address.c

```diff
@@ -87,10 +87,10 @@ void nhdp_decrement_addr_usage(nhdp_addr_t *addr)
     mutex_lock(&mtx_addr_access);
 
     /* Decrement usage count and delete address if no longer used */
-    if (addr) {
+    if (addr && (addr->usg_count > 0)) {
```

Comment:
They are - nevertheless, with parentheses its clear(er) what conditions are checked. This also encouraged by [coding conventions](https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions#indentation-and-braces), otherwise, curly braces are not needed here as well.